### DISCUSSION TITLE
ICICLE Kit Stuck

### DISCUSSION THREAD

**Discussion by fcuzzocrea** [polarfire-soc/polarfire-soc-documentation#21] [Reference designs, FPGA and development kits] [answered]
Hi,

Sorry for for the generic title, but this morning, after switching on my icicle kit I found it to be stuck into a weird state. Nothing is output from the UARTs, and the eFP6 is not recognized by my laptop. I tried on different computers, but with the same result. ICICLE Kit yesterday was working completely fine (I was doing some tests with Yocto).

I can't even flash stock design with Libero, as the eFP6 is not recognized. All the LEDs near raspberry pi connector are on, and LPDDR4 led is off, here is a picture of how it looks my board when I switch it on https://imgur.com/a/WHfmQd8.

Is the second time I face this issue (first time I had to buy a new board, but they are not, let's say, cheap).
Do you have any clue how what is happening and why?

**Comment by Cyril-Jean** [comment]
I am curious. Do you still have the first faulty Icicle Kit. If you did, could you plug it and send a picture of the powered board?

**Comment by fcuzzocrea** [comment]
Yes, I still have it, and behaves the same of this one. Picture of the powered board are available here: https://imgur.com/a/WHfmQd8

**Comment by fcuzzocrea** [comment]
I also tried to reflash it using an external flashpro5 but i did not had any luck.

it seems like the soc does not get powered on. in particular, as you can see from the pictures, the leds corresponding to 1V8 and LPDDR4 are off. Also, the upper-left part of the board gets very hot (I highlighted with a red circle the zone of the board I am talking about in this picture: https://imgur.com/a/gS4pqGS)

**Comment by fcuzzocrea** [comment]
Here is another two pictures of both the boards: https://imgur.com/a/yKEGoXE

**Comment by Cyril-Jean** [comment, marked as answer]
It looks like 2 power regulators got damaged on both boards. Could you reach out to your local friendly Microchip FPGA field engineer to discuss this?

**Comment by fcuzzocrea** [comment]
Thanks, I just contacted him! :)

**Comment by hughbreslin** [comment]
Converting this over to a discussion topic :)
